src/css/controls/flags/cardboard-available.less

```diff
@@ -0,0 +1,14 @@
+.jwplayer.jw-flag-cardboard-available {
+    .jw-icon-cardboard {
+        display: flex;
+    }
+
+    &.jw-state-idle {
```

Comment:
this duplicates the rule in `cast-available.less`. We should remove it from both and put it in `states.less`.